src/vivarium_census_prl_synth_pop/components/population.py

```diff
@@ -173,14 +177,12 @@ def choose_standard_households(self, target_number_sims: int) -> pd.DataFrame:
             weights=standard_households["household_weight"],
         )
 
-        # create unique id for resampled households
+        # create unique id for resampled households -- each census_household_id
+        # can be sampled multiple times.
         chosen_households = pd.DataFrame(
             {
                 "census_household_id": chosen_households,
-                "household_id": np.arange(
-                    data_values.N_GROUP_QUARTER_TYPES,
-                    len(chosen_households) + data_values.N_GROUP_QUARTER_TYPES,
-                ),
+                "census_sample_household_id": np.arange(target_number_sims),
```

Comment:
How is this different from our basic "household_id"?

Reply:
It's not unique with respect to existing household_ids. Only the `Households` component can "mint" new unique household_ids, and it never deletes them, so we don't want to create one for the excess households.

A bit academic here because these are the first households, but this will be consequential for immigration, when there will be pre-existing households we don't want to collide with.